go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/ssvlabs/ssv
 
-go 1.22.6
+go 1.23
```

Comment:
This should be a separate PR

Reply:
It was updated because I just put the latest Go version into the `ssv-signer` repository. I will revert it later, I haven't touched this yet because the PR is still an early draft